### PR TITLE
sysdig-cloud: Cut logs at 100mb and make into a system service

### PR DESCRIPTION
**Pull request by jmreicha**

### CODE REVIEW

File: templates/sysdig-cloud/1/rancher-compose.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      label: "Host exclude label"
+      description: "Specify a Rancher host label here that can be used to exclude deployment of the Sysdig Cloud container on any given host. Eg: sysdig.exclude_sysdigcloud=true (you could then add the label 'sysdig.exclude_sysdigcloud=true' to any Rancher host to exclude Sysdig Cloud from that host). To filter on any of multiple labels, use a comma-separated list."
+      type: "string"
+      default: sysdig.exclude_sysdigcloud=true
+      required: true
+    - variable: "SDC_ADDITIONAL_CONF"
+      label: "Additional Configuration"
+      description: "Content must be valid yaml code, use the escape \n parameter for new lines and adhere to proper indent requirements."
+      type: "string"
+      required: false
```
Comment (rawmind0) [inline templates/sysdig-cloud/1/rancher-compose.yml]: # Line 35 to the end
++    - variable: "log_size"
++      label: "Max log size"
++      description: "Max log size"
++      type: "string"
++      default: 100mb
++      required: true

File: templates/sysdig-cloud/1/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  privileged: true
+  image: sysdig/agent:${VERSION}
+  net: "host"
+  pid: "host"
+  environment:
+    ACCESS_KEY: ${SDC_ACCESS_KEY}
+    TAGS: "${SDC_TAGS}"
+    ADDITIONAL_CONF: "${SDC_ADDITIONAL_CONF}"
+  log_opt:
+    max-size: 100mb
```
Comment (rawmind0) [inline templates/sysdig-cloud/1/docker-compose.yml]: # Line 12
--    max-size: 100mb
++    max-size: ${log_size}

**Comment by rawmind0** [comment]
Hi @jmreicha ...

I request you some minimal changes to be able to set max log size at deploy time. ;)

Thanks...

**Comment by jmreicha** [comment]
@rawmind0 I just pushed the requested changes.  Let me know if it needs anything else.

**Comment by rawmind0** [comment]
Thanks @jmreicha....

But i think you've added some other commits by error....Please review it...

**Comment by jmreicha** [comment]
Oops, I think I accidentally committed the updates to master.  Let me see if I can fix it.

**Comment by jmreicha** [comment]
@rawmind0 I think I fixed it.

**Comment by rawmind0** [comment]
Thanks @jmreicha ...

LGTM..

**Pull request merged**